content/en/apps/guides/hosting/app-developer.md

```diff
@@ -0,0 +1,95 @@
+
+{{% alert title="Note" %}} This guide assumes you are a CHT app developer wanting to either run concurrent instances of the CHT, or easily be able to switch between different instances without loosing any data while doing so. To do development on the CHT core itself, see the [development guide](https://github.com/medic/cht-core/blob/master/DEVELOPMENT.md). 
+
+To deploy the CHT in production, see either [AWS hosting]({{< relref "apps/guides/hosting/self-hosting.md" >}}) or [Self hosting]({{< relref "apps/guides/hosting/ec2-setup-guide.md" >}}){{% /alert %}}
+
+
+## Getting started
+
+Be sure to meet the [CHT hosting requirements]({{< relref "apps/guides/hosting/requirements" >}}) first. As well, if any other `medic-os` instances using [the main `docker-compose.yml` file](https://github.com/medic/cht-core/blob/master/docker-compose.yml) are running locally, stop them otherwise port, storage volume and container name conflicts may occur.
+
+After meeting these requirements, download the developer YAML file in the directory you want to store them:
+
+```shell script
+curl -o docker-compose-developer.yml https://raw.githubusercontent.com/medic/cht-core/master/docker-compose-developer-base.yml
+```
+
+To start the first developer CHT instance, run `docker-compose` and specify the file that was just download:
+
+```shell script
+docker-compose -f docker-compose-developer.yml up
+```
+
+This may take some minutes to fully start depending on the speed of the Internet connection and speed of the bare-metal host.  After it has completed, the CHT should be accessible on [https://localhost](https://localhost).
+
+When connecting to a new dev CHT instance for the first time, an error will be shown, "Your connection is not private" (see [screenshot](/apps/tutorials/local-setup/privacy.error.png)). To get past this, click "Advanced" and then click "Proceed to localhost".
+
+## Running the Nth CHT instance
+
+After running the first instance of the CHT, it's easy to run as many more as are needed.  This is achieved by specifying different:
+
+* port for `HTTP` redirects (`CHT_HTTP`)
+* port for `HTTPS` traffic (`CHT_HTTPS`)
+* project to for the docker compose call (`-p PROJECT`)
+
+Assuming you want to start a new project called `the_second` and  start the instance on `HTTP` port `8081` and `HTTPS` port `8443`, this would be the command:
+
+```shell script
+CHT_HTTP=8081 CHT_HTTPS=8443 docker-compose -p the_second -f docker-compose-developer.yml up
+```
+
+The second instance is now accessible at  [https://localhost:8443](https://localhost:8443).
+
+## The `.env` file 
+
+Often times it's convenient to use revision control, like GitHub, to store and publish changes in a CHT app.  A nice compliment to this is store the specifics on how to run the `docker-compose` command for each app. By using a shared `docker-compose` configuration for all developers on the same app, it avoids any port collisions and enables all developers to have a unified configuration.
```

Comment:
`A nice compliment to this is to store the specifics on how to run the`

Reply:
I'll wait to hear back on how this sentence ends ;) 

otherwise resolving for now to tidy it up so I can send it back to you.

Reply:
Sorry, there was an added to.  It reads a little weird otherwise. 

` A nice compliment to this is store the specifics on how to run the`
 
 `A nice compliment to this is to store the specifics on how to run the`

Reply:
D'oh - That was on me - I totally didn't follow that this was a comment to fix my own content - I thought you were telling me a "nice compliment to this..." :facepalm: Sorry!

Fixed in  c100145 !